### ISSUE TITLE
AlexNet with only  0.4568 accuracy

### ISSUE THREAD

**Issue by liaocs2008** [ColumbiaDVMM/Tamp#2]
I trained AlexNet with your design but only got 0.4568 accuracy. Just want to confirm if you got similar result as me.

**Comment by bjou** [comment]
@Wenri Can you confirm?

**Comment by liaocs2008** [comment]
Great! Actually, I also trained caffenet which got 0.47348.

**Comment by Wenri** [comment]
Yes. I remember that if train the AlexNet directly without changing any parameters, the result would be much lower than reference model. To further improve the result, smaller learning rate should be used. Also more train it for more iterations.

**Comment by liaocs2008** [comment]
Do you mind sharing your configuration files, i.e., prototxts? Your kind assistance on this is very much appreciated. Thanks!

**Comment by Wenri** [comment]
I would like to help you but I don't have time to run the experiments now to check configuration files. I can do this maybe in weekends and I will check back to you later. For your information, Tamp has been changed a lot and I am planning to push it onto GitHub when I can make sure its compatibility with both Caffe and TensorFlow. But since I am the only developer now don't expect this will coming soon and your contribution is highly welcomed.

**Comment by liaocs2008** [comment]
Hi Wenri, did you find the configuration files?

**Comment by Wenri** [comment]
Sorry I did not manage to spare time on it because I was too busy last week. However I am afraid  I might not able to find time until the Reading Week.
